script/test

```diff
@@ -1,32 +1,22 @@
 #!/bin/bash
 
-SCRIPT_DIR=$(dirname "$(readlink -f $0)")
-. ${SCRIPT_DIR}/lib/functions
-
-function cleanup()
-{
-    if [ -z "$CI" ]; then
-        stop_services
-    fi
-}
-
 function do_license_check()
 {
     if [ -n "$CI" ]; then
-        bundle_exec rake license_finder
+        bundle exec rake license_finder
```

Comment:
I'm not sure we should run license_finder with bundler. It calls bundler internally for its purposes.

Reply:
I don't know, it seems correct to me since it's a rake task, and we always run tasks with bundler

Reply:
weshould run license_finder standalone in fact. This rake thing didn't work well in porta. I don't remember all it was.

Since this is no change, it's fine to merge as is.

P.S. haven't looked at the rake task. If it calls `license_finder` as an external command, then it should be fine.

Reply:
Yes, it does:

https://github.com/3scale/apisonator/blob/281309948e882e0d87fd602f27053e5cb1999055/Rakefile#L219-L227